### PR TITLE
Proposing a Community Discussions Page

### PR DESCRIPTION
**Pull request by kariljordan**
Per @ErinBecker "I think it would be nice to have. Maybe it doesn’t need to have it’s own menu pull down item, but could be linked to from the “connect” page?"

The purpose of this page is to provide archived and upcoming community discussions. I'd like to format it similar to https://ropensci.org/commcalls/

### CODE REVIEW

**Comment by kariljordan** [comment]
@tobyhodges @brownsarahm @anelda @jamiehadwin if you have slides from the themed discussion session you presented last year can you add them to this PR?

**Comment by kariljordan** [comment]
@maneesha I am not sure if I put this .md file in the correct place so that it shows up on the website, so once @serahrono reviews feel free to move it.

File: pages/community-discussions.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+[Sign-up](https://pad.carpentries.org/community-discussions)
+
+Thursday February 28    
+23:00 UTC     
+[Click here](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Themed+Discussion+Session%3A+What%27s+New+with+Library+Carpentry&iso=20190228T23&p1=1440&ah=1) for your time zone.    
+[Sign-up](https://pad.carpentries.org/community-discussions)
+
+## Past Calls
+* [What Happens During a Code of Conduct Investigation?](https://docs.google.com/presentation/d/10eLnpfiIjkyZUd9yYlHJAqP_HeQcLJ07L_XAKRzVKiE/edit#slide=id.g3b8317a2f2_1_66) - Karin Lagesen (2019-01-31)    
+* Delivering Exercises to Learners with Load Magic - Sarah Brown (2018-11-7)
```
Comment (brownsarahm) [inline pages/community-discussions.md]: no slides, just a blog post: https://carpentries.org/blog/2018/11/delivering-exercises/

Comment (kariljordan) [reply pages/community-discussions.md]: Thats perfect! I'll add.

File: pages/community-discussions.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+## Community Discussions
```
Comment (fmichonneau) [inline pages/community-discussions.md:8]: the checks on travis are failing because there is no yaml header. Please don't forget to add it :smile:

**Comment by maneesha** [comment]
Added yaml header.
Looks like @kariljordan  would like @serahrono  to review before merging.

**Comment by serahkiburu** [comment]
Thanks for this PR, @kariljordan. Question for you, do we want to embed a calendar at the bottom of the page, with community discussions highlighted in, say, black? That way, 
- we won't need to update the [upcoming events section](https://github.com/carpentries/carpentries.org/blob/e5df423380f8a9350f1974c902254316316bd4b7/pages/community-discussions.md#next-call) of the page every so often, and 
- we can leave a note, [similar to this one](https://github.com/carpentries/carpentries.org/blame/e5df423380f8a9350f1974c902254316316bd4b7/pages/community.md/#L217), explaining what calendar events highlighted in black (or any other user-friendly colour we choose) are.

What do you think?

**Comment by kariljordan** [comment]
@serahrono I love that idea. I was trying to figure out how I would remember to update this page. Can you make the proposed changes so that I can see what it would look like?

**Comment by kariljordan** [comment]
@serahrono checking to see if you're able to add your suggestion to this PR

**Comment by serahkiburu** [comment]
Hey @kariljordan, we'd need to get Calendar Labs enabled by an admin - for me to give this a shot. See https://support.google.com/a/answer/1187241. Maybe @fmichonneau can advise?

**Comment by fmichonneau** [comment]
Calendar labs is being phased out so it'd probably best to not rely on it. Which feature did you need from it? I think that if you change the color of the event on the calendar, they will show in the one displayed on the website.

**Comment by serahkiburu** [comment]
Makes sense, about Calendar Labs @fmichonneau. I don't have permissions to manage these settings on the Community Calendar so I can't test to see that it would work.

**Comment by fmichonneau** [comment]
I elevated your privileges for this calendar. Let me know if it works

**Comment by serahkiburu** [comment]
@kariljordan, here's what the event highlight proposal looks like. screenshot 2019-03-05 at 16 21 25

**Comment by serahkiburu** [comment]
As you can see, because the Community discussions are in the same calendar as other community events (Maintainers' meetings, debriefing sessions, it is not possible to differentiate between these.). My suggestion is that we now consider:
* moving content in the Community Discussions page to this Community Calendar page, as a section under Community Events;
* create a separate calendar with Community Discussions events in it, to allow us to highlight them more prominently in a third, accessibility-friendly colour. We can then embed and display the three calendars as above.

Let me know what you think.

**Comment by kariljordan** [comment]
This is beautiful.

@maneesha which of Serah's recommendations above would be easiest to implement from an infrastructure perspective?

**Comment by tracykteal** [comment]
I like the idea of a separate calendar for Community Discussions, because it seems like it has the added benefit of also making it so people can subscribe just to that calendar. Or it could go on other aligned organizations sites. For instance, I could see rOpenSci adding a link to this calendar on a 'community' type page.

**Comment by kariljordan** [comment]
Let's make a separate calendar for Community Discussions. @serahrono and I will work on this.

**Comment by maneesha** [comment]
I like having a separate Community Discusssion calendar too - agree with what @tracykteal  said above.
Can we create an Asana task for this? I want to be sure to consider other places where we may have calendar integrations so that we don't break things like Zoom room reservations, Asana milestones publications, etc.

**Comment by maneesha** [comment]
Hi all - to be sure I understand things here as you all do...
On [this calendar](https://carpentries.org/community/#community-events) we currently have **all** our public events and milestones.  This includes things like:

* Community calls (pre/post, themed calls, and Carpentries conversations)
* Committee/task force meetings (Maintainers, CarpentryCon, etc.)
* Instructor Training Teaching Demos
* Job plan milestones

While I like the idea of separating calendars out, I need to give more thought on what would make most sense.  I can see the community calls and task force meetings being of general interest, but the teaching demos are likely only of interest to people currently going through instructor training.  

I'd also want to re-think how we are sharing the milestones.  For example, if I go to "Genomics R release" set for March 21, I see all this information.  This is likely to confuse public users when they can't get through the asana link and see tasks in there that don't mean anything to them.

image

I will take some time to explore what we can do to keep the number of calendar options easy to maintain but still allow people to see just what they need to see.  Suggestions are welcome!

**Comment by kariljordan** [comment]
Thanks @maneesha. As we have a lot more themed discussions and Carpentries Conversations coming up in March/April, I'd love to see us resolve this before the 18th if possible. My recommendations:

- Calendar 1: Community Discussions (Pre/Post, Themed Discussions, and #CarpentriesConversations), and Committee/Task Force meetings (Maintainers, CarpentryCon, etc.) that are open to the public.
- Calendar 2: Instructor Training Teaching Demos
- Calendar 3: Job plan milestones, removing Asana links. We could instead link to the job plan for that project (if that's easy to do), or we can talk with @ErinBecker to revisit how we link the milestones.

Thanks all!

File: pages/community-discussions.md (hunk 2 above)
Comment (kariljordan) [reply pages/community-discussions.md:8]: @fmichonneau can you point me to the yaml header? I honestly don't know what that is or where to place it.

Comment (fmichonneau) [reply pages/community-discussions.md:8]: the yaml header was added in a previous commit and all the checks pass now.

**Comment by ErinBecker** [comment]
Thanks for all the thought on this Team! I'd like to suggest that we proceed with splitting the calendars as @serahrono proposed and handle the issue of the Asana Milestones calendar showing irrelevant information as a separate issue. I'll look into that and will update on Asana. Please go ahead and make the other changes independently.

**Comment by maneesha** [comment]
I am merging this as it is for now, as it just reflects stylistic changes to the calendar (week layout and colors).  Separately I'll look to separating out the demo sessions and the task force meetings on a different calendar as there are some access levels I don't want to mess up.

**Pull request merged**